apps/worker/src/app/workflow/usecases/send-message/send-message.usecase.ts

```diff
@@ -206,34 +207,19 @@ export class SendMessage {
     command: SendMessageCommand,
     variables: IFilterVariables
   ): Promise<{ filterResult: IConditionsFilterResponse | null; channelPreferenceResult: boolean | null }> {
-    const skipHalt = this.shouldSkipHalt(bridgeSkip, command.job?.step?.bridgeUrl);
-    if (skipHalt) {
+    if (bridgeSkip === true) {
```

Comment:
We now only skip filter resolution when the Bridge returns `skip: true`, allowing for Global Subscriber and Workflow level Preferences to be evaluated for both stateful _and_ stateless workflows.

Reply:
Yes, we are confident - the `bridge-trigger` tests evaluate both stateful and stateless Workflow execution modes, ensuring that these paths are fully tested.